cfgov/jinja2/v1/know-before-you-owe/index.html

```diff
@@ -0,0 +1,544 @@
+{% extends 'content-base.html' %}
+
+{% block title %}
+    Know Before You Owe: Mortgages &gt; Consumer Financial Protection Bureau
+{% endblock %}
+
+{% block css %}
+<style>
+    .m-hero {
+        background-color: #d6e5e4 !important;
+    }
+
+    .m-hero_image {
+        background-image: url('{{ static("img/kbyo-hero_laptop-2x.png") }}');
+        filter: progid:DXImageTransform.Microsoft.AlphaImageLoader(
+                src='{{ static("img/kbyo-hero_laptop-2x.png") }}',
+                sizingMethod='scale' );
+        padding-bottom: 0;
+    }
+
+    @media screen and (min-width: 37.5625em) {
+        .m-hero_image {
+            background-image: url('{{ static("img/kbyo-hero_laptop-2x.png") }}');
+            filter: progid:DXImageTransform.Microsoft.AlphaImageLoader(
+                    src='{{ static("img/kbyo-hero_laptop-2x.png") }}',
+                    sizingMethod='scale' );
+            padding-bottom: 41.4893617%
```

Comment:
The `{{ super() }}` command needs to move above the `<style>` element in this block so that this `padding-bottom` value will override the default that comes in from `heroes.less`.